.github/workflows/ci.yml

```diff
@@ -1,104 +1,108 @@
 name: CI
 
-on: [push, pull_request, release]
+on: [ pull_request ]
 
 jobs:
   build:
     strategy:
       fail-fast: false
       matrix:
         config:
-        - {
-            name: linux-x64-clang-9,
-            os: ubuntu-18.04,
-            cxx: clang++-9,
-            cmake-build-type: Release
-          }
-        - {
-            name: macos-x64-gcc,
-            os:  macos-10.15,
-            cxx: g++,
-            cmake-build-type: Release
-          }
-        - {
-            name: macos-x64-clang,
-            os:  macos-10.15,
-            cxx: clang++,
-            cmake-build-type: Release
-          }
-        - {
-            name: linux-x64-clang-12-sanitize,
-            os: ubuntu-20.04,
-            cxx: clang++-12,
-            cxx-flags: "-fsanitize=address,undefined",
-            cmake-build-type: Release
-          }
-        - {
-            name: linux-x64-gcc-10-coverage,
-            os: ubuntu-20.04,
-            cxx: g++-10,
-            cxx-flags: --coverage,
-            gcov-tool: gcov-10,
-            cmake-build-type: Debug
-          }
-        - {
-          name: linux-x64-clang-11,
-          os: ubuntu-20.04,
-          cxx: clang++-11,
-          cmake-build-type: Release
-        }
-        - {
-          name: linux-x64-clang-12,
-          os: ubuntu-22.04,
-          cxx: clang++-12,
-          cmake-build-type: Release
-        }
-        - {
-          name: linux-x64-clang-13,
-          os: ubuntu-22.04,
-          cxx: clang++-13,
-          cmake-build-type: Release
-        }
-        - {
-          name: linux-x64-clang-14,
-          os: ubuntu-22.04,
-          cxx: clang++-14,
-          cmake-build-type: Release
-        }
-        - {
-          name: linux-x64-gcc-11,
-          os: ubuntu-22.04,
-          cxx: g++-11,
-          cmake-build-type: Release
-        }
-    name: ${{matrix.config.name}}
+        # These get queued but never actually run
```

Comment:
When merged, we should add issues with tag help-wanted to add windows and macos support to the CI.